src/test/FuzzerImpl.cpp

```diff
@@ -185,23 +185,26 @@ makeAssetCode(int digit)
 }
 
 void
-generateStoredLedgerKeys(StoredLedgerKeys& keys)
+generateStoredLedgerKeys(StoredLedgerKeys::iterator begin,
+                         StoredLedgerKeys::iterator end)
 {
-    static_assert(NUM_UNVALIDATED_LEDGER_KEYS < NUM_STORED_LEDGER_KEYS,
-                  "No room for valid ledger keys");
-
-    for (auto index = 0; index < NUM_VALIDATED_LEDGER_KEYS; ++index)
+    if (std::distance(begin, end) <= NUM_UNVALIDATED_LEDGER_KEYS)
     {
-        keys[index] =
-            LedgerEntryKey(LedgerTestUtils::generateValidLedgerEntry());
+        throw std::runtime_error("No room for unvalidated ledger keys");
     }
 
-    for (auto index = NUM_VALIDATED_LEDGER_KEYS; index < NUM_STORED_LEDGER_KEYS;
-         ++index)
-    {
+    auto const firstUnvalidatedLedgerKey = end - NUM_UNVALIDATED_LEDGER_KEYS;
```

Comment:
You're no longer using `NUM_VALIDATED_LEDGER_KEYS`, so please remove it.

Reply:
Removed.

Reply:
Sorry made a mistake on this one and it was used elsewhere. Please undo when you rebase.

Reply:
I pushed a revert commit so that they'll annihilate when I squash.